demo/index.html

```diff
@@ -14,39 +14,69 @@
   <title>marked-element demo</title>
 
   <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+  <link rel="import" href="../../paper-styles/demo-pages.html">
   <link rel="import" href="../marked-element.html">
+
+  <style is="custom-style">
+
+    .centered {
+      max-width: 550px;
+      padding: 0;

```

Comment:
I think you only need the `max-width`, since the rest is defined in `demo-pages`. Is 550px too much for mobile?